src/ariadne_wallhug.c

```diff
@@ -2051,7 +2051,14 @@ TbBool slab_good_for_computer_dig_path(const struct SlabMap *slb)
     return false;
 }
 
-static TbBool is_valid_hug_subtile(MapSubtlCoord stl_x, MapSubtlCoord stl_y, PlayerNumber plyr_idx)
+TbBool slab_good_for_computer_claim_path(const struct SlabMap* slb)
```

Comment:
This new function is near identical to the already existing `TbBool slab_kind_is_liquid()`, so delete this new function.

Use `slab_kind_is_liquid(slb->kind)` instead of `slab_good_for_computer_claim_path(slb)` to call on the functionality you desire.